charts/telepresence-oss/README.md

```diff
@@ -30,13 +30,12 @@ The following tables lists the configurable parameters of the Telepresence chart
 | agent.image.registry                                 | The registry for the injected agent image                                                                                                                           | `ghcr.io/telepresenceio`                                                    |
 | agent.initResources                                  | The resources for the injected init container                                                                                                                       |                                                                             |
 | agent.logLevel                                       | The logging level for the traffic-agent                                                                                                                             | defaults to logLevel                                                        |
-| agent.maxIdleTime                                    | maximum time the agent is idle (no engagements to the workload) before it is cleaned up by the traffic manager                                                      |                                                                             |                                                                                   "0h (infinite, never cleaned up)"
+| agent.maxIdleTime                                    | maximum time the agent is idle (no engagements to the workload) before it is cleaned up by the traffic manager                                                      | `0h` (infinite, never cleaned up)                                           |
```

Comment:
@Yuheng3107 you introduced that parameter in #3903, I edited a little bit the formatting of the description for the parameter, can you confirm this PR reflects your intent?